cmd/snap/cmd_first_boot.go

```diff
@@ -40,7 +41,7 @@ func init() {
 }
 
 func (x *cmdInternalFirstBoot) Execute(args []string) error {
-	err := snappy.FirstBoot()
+	err := overlord.FirstBoot()
```

Comment:
just return overlord.FirstBoot() given we are not returning the special error, anyway all this is going away so i18n not so important